packages/squiggle-lang/__tests__/TestHelpers.res

```diff
@@ -1,6 +1,16 @@
 open Jest
 open Expect
 
+let expectErrorToBeBounded = (received, expected, ~epsilon) => {
+  let distance = Js.Math.abs_float(received -. expected)
+  let error = if expected < epsilon ** 2.5 {
```

Comment:
where did 2.5 come from?

Reply:
this is not well thought out. I was simply deciding between `2` and `3` and decided on `2.5`. 

Without this `if` block, I was getting `Infinity` when `expected` would be too close to zero.

Reply:
(realizing it should be `Js.abs_float(expected)` anyway)

Reply:
`epsilon ** 2.5` is actually doubly bad because in practice I found myself passing in large (`>1`) epsilon! so this is _not_ good code _at all_.

Reply:
I can also take a `max` of some small but known number and the passed in `expected`, for a more concise way of getting what I want.

Reply:
@OAGr see recent commit.

Reply:
Even with the new one, it still feels weird. Can you at least leave a comment explaining this? Comments are really especially important to have, directly in the codebase, for weird issues.

Reply:
This function warrants some fairly intense testing. I'm uncertain if we should do that before it goes into `develop` at all.

Reply:
I'm going to go ahead now. This PR is already one refactor. We could have another later.